### PR TITLE
Adding GDPR boxes to GET sign in and LIST sign in 

### PR DESCRIPTION
**Pull request by besiler**
Privacy let us know that we need a GDPR statement since the API can be used to export personal data. Added that statement as approved by Eddie Young (CELA) and Peter Gurevich (Privacy)

### CODE REVIEW

**Comment by besiler** [comment]
@FaithOmbongi  you're copied on the email thread with CELA - subject line is "[CELA guidance needed] Adding GDPR label to Azure AD reporting docs"

Please let me know if you need any info/approvals!

**Review by isvargasmsft — APPROVED** [review]
@FaithOmbongi FYI

**Comment by besiler** [comment]
Marked ready to merge since review is done

File: api-reference/beta/api/signin-get.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ Namespace: microsoft.graph
 
 Get a [signIn](../resources/signin.md) object that contains a specific user sign-in event for your tenant. This includes sign-ins where a user is asked to enter a username or password, and session tokens.
 
+> [!NOTE]
+> This article provides steps about how to export personal data from the device or service and can be uses to support your obligations under the GDPR. You can correct, update, or delete identifiable information about end users, such as customer and employee user profiles and user work information that contain personal data, such as a user's name, work title, address, or phone number, in your [Azure Active Directory (Azure AD)](https://azure.microsoft.com/services/active-directory/) environment by using the [Azure portal](https://portal.azure.com/). You must sign in with an account that's a global admin for the directory.
```
Comment (Lauragra) [inline api-reference/beta/api/signin-get.md:19]: Let's revise this note text to be a bit more readable and concise. Suggested revision:

This article describes how to export personal data from a device or service. These steps can be used to support your obligations under the General Data Protection Regulation (GDPR). You can use the [Azure portal](https://portal.azure.com/) to correct, update, or delete identifiable information about end users, including customer and employee user profiles or personal data, such as a user's name, work title, address, or phone number, in your [Azure Active Directory (Azure AD)](https://azure.microsoft.com/services/active-directory/) environment . You must have a global admin account for the directory.

**Comment by FaithOmbongi** [comment]
Flipping this to **do not merge** to allow CELA to review and approve changes to the note.

**Comment by besiler** [comment]
Thanks @FaithOmbongi !

**Pull request merged**